--- Makefile	2023-09-29 11:39:43.149260825 +0000
+++ Makefile_new	2024-02-29 17:17:37.528243528 +0000
@@ -15,9 +15,9 @@
 #------------------------------------------------------------------------
 # things that are specific to this contrib
 NAME=RecursiveTools
-SRCS=Recluster.cc RecursiveSymmetryCutBase.cc ModifiedMassDropTagger.cc SoftDrop.cc IteratedSoftDrop.cc RecursiveSoftDrop.cc BottomUpSoftDrop.cc
+SRCS=Recluster.cc RecursiveSymmetryCutBase.cc ModifiedMassDropTagger.cc SoftDrop.cc IteratedSoftDrop.cc RecursiveSoftDrop.cc BottomUpSoftDrop.cc Util.cc
 EXAMPLES=example_mmdt example_mmdt_sub example_mmdt_ee example_recluster example_softdrop example_recursive_softdrop example_advanced_usage example_isd example_bottomup_softdrop
-INSTALLED_HEADERS=Recluster.hh RecursiveSymmetryCutBase.hh ModifiedMassDropTagger.hh SoftDrop.hh IteratedSoftDrop.hh RecursiveSoftDrop.hh BottomUpSoftDrop.hh
+INSTALLED_HEADERS=Recluster.hh RecursiveSymmetryCutBase.hh ModifiedMassDropTagger.hh SoftDrop.hh IteratedSoftDrop.hh RecursiveSoftDrop.hh BottomUpSoftDrop.hh Util.hh
 #------------------------------------------------------------------------
 
 CXXFLAGS+= $(shell $(FASTJETCONFIG) --cxxflags)
